common/static/js/src/utility.js

```diff
@@ -37,3 +37,17 @@ window.rewriteStaticLinks = function(content, from, to) {
     var regex = new RegExp('(https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}([-a-zA-Z0-9@:%_\+.~#?&//=]*))?' + fromRe, 'g');
     return content.replace(regex, replacer);
 };
+
+window.rewriteCdnLinksToStatic = function(content) {
+    var to = 'static'
+    if (to === null) {
```

Comment:
There is no need for this `if` condition as we are explicitly setting the value of `to` variable.

Reply:
Alright.